models/repo/repo_list.go

```diff
@@ -688,6 +688,24 @@ func FindUserCodeAccessibleRepoIDs(user *user_model.User) ([]int64, error) {
 	return repoIDs, nil
 }
 
+// FindUserCodeAccessibleUserRepoIDs finds all at Code level accessible repositories' ID by the user's id with specific owner.
+func FindUserCodeAccessibleUserRepoIDs(ownerID int64, user *user_model.User) ([]int64, error) {
```

Comment:
I think @zeripath would like a `ctx context.Context` here...
```suggestion
func FindUserCodeAccessibleUserRepoIDs(ctx context.Context, ownerID int64, user *user_model.User) ([]int64, error) {
```

Reply:
Also, that name is only marginally good. What about `FindCodeAccessibleRepoIDs` instead?

Reply:
That would be too generic and would conflict with function that there is already named `FindUserCodeAccessibleRepoIDs`

Reply:
renamed to `FindUserCodeAccessibleOwnerRepoIDs`